.github/workflows/test.yml

```diff
@@ -17,17 +17,26 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.6', '3.7', '3.8', '3.9', '3.10']
+        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
```

Comment:
this requires an update to the changelog and a major version bump in `smile_id_core/__init__.py`